src/fileRendering.ts

```diff
@@ -17,14 +18,21 @@ export function copyFilesDir(filesBasePath: string, context: NetworkContext): vo
         const outputPath = resolvePath(context.outputPath, filePath);
         const outputDirname = dirname(outputPath);
 
-        const mode = fs.statSync(resolvePath(filesBasePath, filePath)).mode;
-        const fileSrc = fs.readFileSync(resolvePath(filesBasePath, filePath), "utf-8");
-        const output = fileSrc.replace(/(\r\n|\n|\r)/gm, os.EOL);
+        const { mode, size } = fs.statSync(resolvePath(filesBasePath, filePath));
 
         if (!validateDirectoryExists(outputDirname)) {
             fs.mkdirSync(outputDirname, { recursive: true });
         }
 
+        if (isBinaryFileSync(resolvePath(filesBasePath, filePath), size)) {
+            fs.createReadStream(resolvePath(filesBasePath, filePath)).pipe(fs.createWriteStream(outputPath, {
+              mode,
+            }));
+            continue
```

Comment:
`continue` and `break` are considered code smells unless you're using them to verify some precondition at the top of a loop. `continue` in this case acts as a sort of trap and creates a secondary exit in the middle of the loop, where a reader who's just scanning through might not expect it. With `if/else` here instead of `continue`, you can more clearly see the control flow and determine the cyclomatic complexity. It also makes the function simpler to edit in the future, as there's only one clear entrance and exit path.

Reply:
If you think that I'm being pedantic here, bear in mind that GitHub even uses indented blocks to determine how much context to give you in the diff display when reviewing PRs. For example, have a look at your changes down in `src/networkBuilder.ts` in side-by-side diff mode. See how in the old version it shows you lines 21 and 22, but collapses the branch that you didn't modify?

Reply:
Okay, it looks like you're using the same `docker-compose` as the other two code paths now. That's good, as it'll allow us to enable private transactions in the orchestrate code path as well.

Reply:
I don't have any strong preference agains usage of `continue` against  `if/else` statements, although as personal preference I tend to use early exits and loop breaks instead of big if/else block :)

Setting aside code implementation details this code is revered in my next commit as the binary leading to this it is not part of Orchestrate files anymore